src/lib/registration/ui/addon_selection_reregistration_dialog.rb

```diff
@@ -62,6 +64,8 @@ def addon_selected?(addon)
       # empty implementation, allow reregistration of a dependant addon
       # without reregistering its parent
       def reactivate_dependencies
+        # FIXME: how to test this?
```

Comment:
I do not get this fixme, what needed to test?
also comment above is no longer correct